niworkflows/interfaces/cifti.py

```diff
@@ -15,47 +16,62 @@
     'CIFTI_STRUCTURE_CORTEX_LEFT': None,
     'CIFTI_STRUCTURE_CORTEX_RIGHT': None,
 
     # SUBCORTICAL
-    'CIFTI_STRUCTURE_ACCUMBENS_LEFT': [26],
-    'CIFTI_STRUCTURE_ACCUMBENS_RIGHT': [58],
-    'CIFTI_STRUCTURE_AMYGDALA_LEFT': [18],
-    'CIFTI_STRUCTURE_AMYGDALA_RIGHT': [54],
-    'CIFTI_STRUCTURE_BRAIN_STEM': [16],
-    'CIFTI_STRUCTURE_CAUDATE_LEFT': [11],
-    'CIFTI_STRUCTURE_CAUDATE_RIGHT': [50],
-    'CIFTI_STRUCTURE_CEREBELLUM_LEFT': [6],
-    'CIFTI_STRUCTURE_CEREBELLUM_RIGHT': [45],
-    'CIFTI_STRUCTURE_DIENCEPHALON_VENTRAL_LEFT': [28],
-    'CIFTI_STRUCTURE_DIENCEPHALON_VENTRAL_RIGHT': [60],
-    'CIFTI_STRUCTURE_HIPPOCAMPUS_LEFT': [17],
-    'CIFTI_STRUCTURE_HIPPOCAMPUS_RIGHT': [53],
-    'CIFTI_STRUCTURE_PALLIDUM_LEFT': [13],
-    'CIFTI_STRUCTURE_PALLIDUM_RIGHT': [52],
-    'CIFTI_STRUCTURE_PUTAMEN_LEFT': [12],
-    'CIFTI_STRUCTURE_PUTAMEN_RIGHT': [51],
-    'CIFTI_STRUCTURE_THALAMUS_LEFT': [10],
-    'CIFTI_STRUCTURE_THALAMUS_RIGHT': [49],
+    'CIFTI_STRUCTURE_ACCUMBENS_LEFT': (26,),
+    'CIFTI_STRUCTURE_ACCUMBENS_RIGHT': (58,),
+    'CIFTI_STRUCTURE_AMYGDALA_LEFT': (18,),
+    'CIFTI_STRUCTURE_AMYGDALA_RIGHT': (54,),
+    'CIFTI_STRUCTURE_BRAIN_STEM': (16,),
+    'CIFTI_STRUCTURE_CAUDATE_LEFT': (11,),
+    'CIFTI_STRUCTURE_CAUDATE_RIGHT': (50,),
+    'CIFTI_STRUCTURE_CEREBELLUM_LEFT': (
+        6,  # DKT31
+        8,  # HCP MNI152
+    ),
+    'CIFTI_STRUCTURE_CEREBELLUM_RIGHT': (
+        45,  # DKT31
+        47,  # HCP MNI152
+    ),
+    'CIFTI_STRUCTURE_DIENCEPHALON_VENTRAL_LEFT': (28,),
+    'CIFTI_STRUCTURE_DIENCEPHALON_VENTRAL_RIGHT': (60,),
+    'CIFTI_STRUCTURE_HIPPOCAMPUS_LEFT': (17,),
+    'CIFTI_STRUCTURE_HIPPOCAMPUS_RIGHT': (53,),
+    'CIFTI_STRUCTURE_PALLIDUM_LEFT': (13,),
+    'CIFTI_STRUCTURE_PALLIDUM_RIGHT': (52,),
+    'CIFTI_STRUCTURE_PUTAMEN_LEFT': (12,),
+    'CIFTI_STRUCTURE_PUTAMEN_RIGHT': (51,),
+    'CIFTI_STRUCTURE_THALAMUS_LEFT': (10,),
+    'CIFTI_STRUCTURE_THALAMUS_RIGHT': (49,),
+}
+CIFTI_VARIANTS = {
+    'HCP grayordinates': ('fsLR', 'MNI152NLin6Asym'),
+    'space1': ('fsaverage5', 'MNI152NLin2009cAsym'),
+    'space2': ('fsaverage6', 'MNI152NLin2009cAsym'),
 }
 
 
 class _GenerateCiftiInputSpec(BaseInterfaceInputSpec):
     bold_file = File(mandatory=True, exists=True, desc="input BOLD file")
-    volume_target = traits.Enum("MNI152NLin2009cAsym", mandatory=True, usedefault=True,
+    volume_target = traits.Enum("MNI152NLin6Asym", "MNI152NLin2009cAsym", usedefault=True,
                                 desc="CIFTI volumetric output space")
-    surface_target = traits.Enum("fsaverage5", "fsaverage6", mandatory=True,
-                                 usedefault=True, desc="CIFTI surface target space")
+    surface_target = traits.Enum("fsLR", "fsaverage5", "fsaverage6", usedefault=True,
+                                 desc="CIFTI surface target space")
+    density = traits.Enum(None, '32k', '59k', usedefault=True,
+                          help='surface hemisphere vertices')
```

Comment:
Please define what `None` means in the desc (BTW, the appropriate argument name is `desc`).

```suggestion
                          desc='surface hemisphere vertices')
```